tests/magnetostatics/test_circuits.py

```diff
@@ -294,6 +296,40 @@ def test_field_plot(self):
         plt.show()
 
 
+class TestPolyhedralFaceContinuity:
+    @classmethod
+    def setup_class(cls):
+        shape = make_circle(radius=10, axis=(0, 1, 0))
+        xs = Coordinates({"x": [-1, -1, 1, -1], "z": [-1, 1, 0, -1]})
+        xs.translate(xs.center_of_mass)
+
+        cls.circuit = ArbitraryPlanarPolyhedralXSCircuit(
+            shape.discretize(ndiscr=15), xs, current=1e6
+        )
+
+    def test_field_faces(self):
+        point = np.array([2, 2, 2])
+        source1 = self.circuit.sources[4]
+        source2 = self.circuit.sources[5]
+        field1 = source1._rho * _field_fabbri(
+            source1._dcm[1],
+            np.array([source1._face_points[-1]]),
+            np.array([source1._face_normals[-1]]),
+            np.array([source1._mid_points[-1]]),
+            np.array([3]),
+            point,
+        )
+        field2 = source2._rho * _field_fabbri(
+            source2._dcm[1],
+            np.array([source2._face_points[0]]),
+            np.array([source2._face_normals[0]]),
+            np.array([source2._mid_points[0]]),
+            np.array([3]),
+            point,
+        )
+        assert np.allclose(field1, -field2, rtol=0, atol=EPS)
```

Comment:
@kj5248 yeah... still don't know what to do about this.

Reply:
I think maybe just need to remove this part as its not true for this method? Especially as have the test that compares 2 half shapes to a full shape?